examples/esbuild/package.json

```diff
@@ -7,6 +7,9 @@
         "tslib": "1.9.0",
         "typescript": "3.5.3"
     },
+    "dependencies": {
+        "chalk": "4.1.0"
```

Comment:
Should a package-lock.json also be changing...?

Reply:
yeah looks like we should be using `npm ci` when we install deps for examples

Reply:
Nice catch @jbedard, I should have regenerated the lock file, will update. 
Currently we can't use `npm ci` during install of the deps in integration tests as the package replacement gets in the way, or is that not what you meant?